mmv1/products/firebaseappcheck/firebase_app_check_service_config.yaml

```diff
@@ -0,0 +1,117 @@
+self_link: projects/{{project}}/services/{{service_id}}
+create_verb: :PATCH
+update_verb: :PATCH
+update_mask: true
+delete_verb: :PATCH
+description: The enforcement configuration for a service supported by App Check.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://firebase.google.com/docs/app-check'
+  api: 'https://firebase.google.com/docs/reference/appcheck/rest/v1/projects.services'
+import_format:
+  [
+    'projects/{{project}}/services/{{service_id}}',
+    '{{project}}/{{service_id}}',
+    '{{service_id}}',
+  ]
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  custom_create: templates/terraform/custom_create/firebase_app_check_service_config.go
+  custom_delete: templates/terraform/custom_delete/firebase_app_check_service_config.go
+  test_check_destroy: templates/terraform/custom_check_destroy/firebase_app_check_service_config.go
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'firebase_app_check_service_config_off'
+    primary_resource_id: 'default'
+    vars:
+      service_id: "firestore.googleapis.com"
+    test_vars_overrides:
+      service_id: "\"firestore.googleapis.com\""
+    test_env_vars:
+      project_id: :PROJECT_NAME
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'firebase_app_check_service_config_enforced'
+    primary_resource_id: 'default'
+    vars:
+      service_id: "firebasestorage.googleapis.com"
+    test_vars_overrides:
+      service_id: "\"firebasestorage.googleapis.com\""
+    test_env_vars:
+      project_id: :PROJECT_NAME
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'firebase_app_check_service_config_unenforced'
+    primary_resource_id: 'default'
+    vars:
+      service_id: "firebasedatabase.googleapis.com"
+    test_vars_overrides:
+      service_id: "\"firebasedatabase.googleapis.com\""
+    test_env_vars:
+      project_id: :PROJECT_NAME
+parameters:
+  - !ruby/object:Api::Type::String
+    name: service_id
+    description: |
+      The identifier of the service to configure enforcement. Currently, the following service IDs are supported:
+        firebasestorage.googleapis.com (Cloud Storage for Firebase)
+        firebasedatabase.googleapis.com (Firebase Realtime Database)
+        firestore.googleapis.com (Cloud Firestore)
+        identitytoolkit.googleapis.com (Authentication)
+    required: true
+    immutable: true
+    url_param_only: true
+properties:
+  - !ruby/object:Api::Type::String
+    name: name
+    description: |
+      The fully-qualified resource name of the service enforcement configuration.
+    output: true
+    pattern: projects/{{project}}/services/{{service_id}}
+  - !ruby/object:Api::Type::Enum
+    name: enforcementMode
+    description: |
+      The App Check enforcement mode for a service supported by App Check. Valid values are
+
+      (Unset)
+      Firebase App Check is not enforced for the service, nor are App Check metrics collected.
```

Comment:
nit: We may want to note that leaving the field unset is equivalent to sending an explicit `OFF`

Reply:
In particular, we should note the existence of the `OFF` enum value being the default zero value.